src/app/feature/consultant/services/consultant-store/consultant-store.service.ts

```diff
@@ -136,6 +138,10 @@ export class ConsultantStore implements OnDestroy {
         .subscribe(consultant => {
           this.router.navigate([ '/consultant'], { queryParams: { id: consultant.id } });
         });
+        this.notification.notificationsBar('Consultant Profile Saved Successfully!', 'success');
+    }
+    else {
+      this.notification.notificationsBar('There was an error saving consultant', 'error');
```

Comment:
In theory this should never be called. I added the conditional just in case someone tried to call addNewConsultant when the store was configured for an existing consultant